src/multi_period_dev.py

```diff
@@ -615,7 +615,12 @@ def solve_multi_period_fpl(data, options):
     if options.get("no_transfer_gws", None) is not None:
         if len(options['no_transfer_gws']) > 0:
             model.add_constraint(so.expr_sum(transfer_in[p,w] for p in players for w in options['no_transfer_gws']) == 0, name='banned_gws_for_tr')
-    
+
+    if options.get('no_transfer_by_position', None) is not None:
+        if len(options['no_transfer_by_position']) > 0:
+            # ignore w=1 as you must transfer in a full squad
+            model.add_constraint(so.expr_sum(transfer_in[p,w] for p in players if merged_data.loc[p, 'Pos'] in options['no_transfer_by_position'] for w in gameweeks if w > 1) == 0, name='no_transfer_by_position')
```

Comment:
I think this might cause an issue when wildcarding. Perhaps it should be <= use_wc[w] ?

Reply:
I think it's up for interpretation, I assumed that if you turn off the position group it would be turned off for WC too. But I see the argument for it not being relevant as the topic is probably around "wasting" FTs not being locked in on a particular position group

Reply:
Yeah I feel like this will cause an issue with people keeping this in their settings, and then trying to wildcard. Thanks for the contribution.